adm/src/database/lmdb.rs

```diff
@@ -217,3 +217,153 @@ impl<'a> LmdbDatabaseWriter<'a> {
             DatabaseError::WriterError(format!("{}", err)))
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Consider putting mods in their own file

Reply:
I think `cargo test` requires this arrangement.

Reply:
Not necessarily, I think you can have:

```
#[cfg(test)]
mod test_lmdb;
```
and then put the files in a `test_lmdb.rs` file, I think.

I'm fine with leaving this as is, though